routers/user/setting/account.go

```diff
@@ -132,7 +132,7 @@ func EmailPost(ctx *context.Context) {
 				ctx.Redirect(setting.AppSubURL + "/user/settings/account")
 				return
 			}
-			mailer.SendActivateEmailMail(ctx.Locale, ctx.User, email)
+			mailer.SendActivateEmailMail(ctx.User, email)
```

Comment:
since its NOT primary email, we use ctx.User's language because we guess an activated account is already setup and so the lang should be stored